website/docs/docs/dbt-cloud-apis/project-state.md

```diff
@@ -83,3 +83,21 @@ Here are some limitations of the state representation in the Discovery API:
 - Compiled code results may be outdated depending on dbt Cloud run step order and failures.
 - Catalog info can be outdated, or incomplete (in the applied state), based on if/when `docs generate` was last run.
 - Source freshness checks can be out of date (in the applied state) depending on when the command was last run, and it’s not included in `build`. 
+
+
+## Supported features for applied state
+
+The following lists the features available for adapters:
+
+| Adapter | Catalog | Source freshness |
+|---------|---------|------------------|
+| `dbt-snowflake` | incremental | metadata-based |
```

Comment:
`dbt-bigquery`, `dbt-redshift`, and `dbt-databricks` all have the same support as `dbt-snowflake`.
`dbt-postgres` has incremental support for Catalog and `loaded_at` field support for Source freshness.
It might be worth adding a row at the bottom for "default", which would match `dbt-spark`.

Reply:
FYI, I think we're about to remove incremental catalog support from `dbt-databricks`. So this might change today.

Reply:
We merged the `dbt-databricks` change, so they are now a manual run for catalog.

Reply:
thanks for the heads up! will update and should have a new draft my tomorrow morning

Reply:
Wanna push a commit @nghi-ly ?

Then hopefully we can merge after that?